index.js

```diff
@@ -1,29 +1,30 @@
-
-const core   = require('@actions/core');
+const core = require('@actions/core');
 //const github = require('@actions/github');
 const semver = require('semver');
 
 try {
-	const since_perl = semver.coerce (core.getInput('since-perl'));
-	const with_devel = core.getInput('with-devel') == "true";
+    const since_perl = semver.coerce(core.getInput('since-perl'));
```

Comment:
please retain indent with tabs. That is something I call "contributor friendly", allowing everyone to set their own tab width to whatever they want, letting tab representing "1 indent level"

Reply:
I've added `pretty` and this is auto converted by `npm run pretty`